src/mutation_input_structs.rs

```diff
@@ -0,0 +1,40 @@
+use async_graphql::{InputObject, SimpleObject};
+use bson::Uuid;
+use std::{
+    cmp::Ordering,
+    collections::{BTreeSet, HashSet},
+};
+
+#[derive(SimpleObject, InputObject)]
+pub struct CreateOrderInput {
+    /// UUID of user owning the order.
+    pub user_id: Uuid,
+    /// OrderItems of order.
+    pub order_items: BTreeSet<OrderItemInput>,
+}
+
+#[derive(SimpleObject, InputObject, PartialEq, Eq)]
+pub struct OrderItemInput {
```

Comment:
missing the "how many do you want" thing

Reply:
Will be queried now